x-pack/plugins/cases/common/constants.ts

```diff
@@ -55,6 +55,8 @@ export const CASE_USER_ACTIONS_URL = `${CASE_DETAILS_URL}/user_actions`;
 export const CASE_ALERTS_URL = `${CASES_URL}/alerts/{alert_id}`;
 export const CASE_DETAILS_ALERTS_URL = `${CASE_DETAILS_URL}/alerts`;
 
+export const CASE_DETAILS_METRICS_URL = `${CASE_DETAILS_URL}/metrics`;
```

Comment:
Good idea. That sounds good to me.

Reply:
I like the idea of isolating all the `/metrics` API, but maybe the `/case` part at the end is a little redundant? will we need metrics for other entities rather than cases?
Won't:
```
/api/cases/metrics- metrics for all cases
/api/cases/metrics/<case_id> - metrics for a specific case
```
be enough?

Reply:
> will we need metrics for any entity other than cases?

I don't foresee any at the moment but it might be nice to have the option to add more specific metric APIs in the future 🤷‍♂️